data/core/encyclopedia/geography.cfg

```diff
@@ -170,7 +171,7 @@ Yet despite the sheer wilderness above the Northlands, the subterranean Dwarvish
         _ "<header>text='Notable land features:'</header>
     • <ref>dst='heart_mountains' text='Heart Mountains'</ref>: A virtually impassable barrier between the river country and the Northern Plains.
     • Heartfangs: The particularly forbidding stretch of high peaks southwest of Lake Vrug and north of the Forest of Wesmere. The most inhospitable and dangerous portion of the Heart Mountains; only hermits, madmen, and mages live there.
-    • Swamp of Dread: A very large bog located between the Heart Mountains and the Great River. A notoriously dangerous place.
+    • Swamp of Dread: A very large bog located between the Heart Mountains and the Great River. A notoriously dangerous place filled with violent saurians.
```

Comment:
I don't think "violent" is a very good word to use to describe saurians…

Reply:
They are said to be very violent in the unit description, and form the perspective of Wesnoth they would likely be considered quite barbaric, even if the unit description is exaggerated. However, I should probably choose a better adjective to convey that.